.travis.yml

```diff
@@ -1,20 +1,14 @@
 language: node_js
 node_js:
-  - 8
   - 10
+  - 12
 
 env:
   global:
-  - SAUCE_USERNAME: vojtajina
-  - secure: "k2x1puIrj42LqnWF8SoDod19xpeJhJN3RDMgRLK2LTYffcdJV8TXilPRUk4pXyN6u8B79fxAW8WucsrlxB6CaaB5cXsFTDG+jpO21AlFc3mFjPh6sTkp/1L/bxc3cJNc9w18Ol2+8MO5s37xuy/lJaQeBRFxQleZZWCLLjtXytg="
+  - SAUCE_USERNAME: karmarunnerbot
```

Comment:
Maybe configure `SAUCE_USERNAME` in Travis UI as well and remove the whole section?